.github/workflows/build-mysql-container.yaml

```diff
@@ -15,15 +15,50 @@ on:
       - "!**.md"
 
 jobs:
+  filter:
+    runs-on: ubuntu-20.04
+    outputs:
+      mysql-versions: ${{ steps.filter.outputs.mysql-versions }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: filter
+        id: filter
+        working-directory: containers
+        run: |
+          array=()
+
+          for dir in $(find ./mysql -mindepth 1 -type d -printf '%f\n'); do
+            result=$(./tag_exists moco/mysql "mysql/$dir")
+
+            if [ "$result" = 'ng' ]; then
+              array+=( "$dir" )
+            fi
+
+          done
+
+          if [ "${#array[@]}" -eq 0 ]; then
+            json_output="[]"
+          else
+            json_output=$(printf '%s\n' "${array[@]}" | jq -R . | jq -s . | jq -c .)
+          fi
+
+          echo "GITHUB_OUTPUT: mysql-versions=$json_output"
+          echo "mysql-versions=$json_output" >> "$GITHUB_OUTPUT"
```

Comment:
Previously, Step "Check TAG file" would fail the job if the image already existed.
This caused a bug that caused GitHub Actions jobs to fail if the tag file contained some unchanged mysql versions.

I added filter jobs to build only mysql images that have changed

Reply:
I tested jobs:

refs: https://github.com/cybozu-go/moco/actions/runs/5664698208?pr=542

Reply:
Good changes!

So don't we need to run `tag_exists` in the `Check TAG file` step? Or it is better to leave it?

Reply:
Thank you very much!You're right, the double tag file check was unnecessary, so I removed the duplicate process.
https://github.com/cybozu-go/moco/pull/542/commits/70415756959286fc0de1c6825add6f6a66074ca4